app/src/main/java/com/infomaniak/mail/data/cache/mailboxContent/MessageController.kt

```diff
@@ -215,6 +214,8 @@ object MessageController {
 
         messages.forEach { message ->
 
+            if (!message.seen) incrementFolderUnreadCount(message.folderId, 1)
```

Comment:
The `folderId` can be different for each Message.

We'll probably have to loop across a `mapOf<folderId, unreadCount>` after this `messages.forEach`.
I'm not so sure it will be simpler.

Reply:
Okay i see, so i agree with this case then.